assignment_3/Area/src/Area.java

```diff
@@ -0,0 +1,160 @@
+import java.util.Scanner;
+
+
+public class Area 
+{
+	/**
+	 * 
+	 * @param base of triangle
+	 * @param height of triangle
+	 * @return Area of triangle
+	 */
+	
+	public double Area_of_Triangle(double triangle_base,double triangle_height)
+	{
+		double Area_Triangle;
+		Area_Triangle= 0.5*triangle_base*triangle_height;
+		return Area_Triangle;
+	}
+	
+	
+	/**
+	 * 
+	 * @param width of rectangle
+	 * @param height of rectangle
+	 * @return Area of rectangle
+	 */
+
+	public double Area_of_Rectangle(double width_rectangle, double height_rectangle)
+	{
+		double Area_Rectangle=width_rectangle*height_rectangle;
+		return Area_Rectangle;
+	}
+	
+	/**
+	 * 
+	 * @param side of square
+	 * @return area of square
+	 */
+	
+	public double Area_of_Square(double side_square)
+	{
+		double Area_square=side_square*side_square;
+		return Area_square;
+	}
+	
+	/**
+	 * 
+	 * @param radius of circle
+	 * @return Area of circle
+	 */
+	public double Area_of_Circle(double radius_circle)
+	{
+		
+		double Area_circle=(22/7)*radius_circle*radius_circle;
```

Comment:
Area or circle will not return expected output.
(22/7) = 3 quotient
It should be 22.0/7 or 3.14
or radiusofcircle * radiusofcircle * 22 / 7;